internal/data/daily.go

```diff
@@ -18,31 +19,36 @@ type DailyDownload struct {
 	Downloads int    `json:"downloads"`
 }
 
-func GetDaily(period, pkg string) (Daily, error) {
+func GetDaily(period Period, pkgs []string) ([]Daily, error) {
 	// weirdly the API returns an array of length 1
 	var daily []Daily
-
-	path := URL + "downloads/daily/" + period + "/" + pkg
-
+	path := URL + "downloads/daily/" + string(period) + "/" + strings.Join(pkgs, ",")
 	resp, err := http.Get(path)
 
 	if err != nil {
-		return Daily{}, err
+		return daily, err
+	}
+	// This is currently kind of meaningless until
+	// https://github.com/r-hub/cranlogs.app/issues/41 is resolved but at least
+	// its there
+	if resp.StatusCode != 200 {
+		return daily, err
 	}
-
 	defer resp.Body.Close()
 
 	body, err := ioutil.ReadAll(resp.Body)
 
 	if err != nil {
-		return Daily{}, err
+		return daily, err
 	}
 
 	err = json.Unmarshal(body, &daily)
-
+	// if this errors given the current 200 status code returning an error response
```

Comment:
this is now (partially) fixed due to the new deployment that does 404 (at least for some cases)